ibis/expr/datatypes.py

```diff
@@ -698,6 +698,7 @@ class MultiPolygon(GeoSpatial):
 interval = Interval()
 category = Category()
 # geo spatial data type
+geospatial = GeoSpatial()
```

Comment:
It doesn't make much sense to expose it in the API, I think We should remove it.